test/behaviour/resolution/ResolutionSteps.java

```diff
@@ -97,6 +97,11 @@ public void reasoned_keyspace_all_answers_are_correct() {
         resolution.testResolution(queryToTest);
     }
 
+    @Then("no answers are resolved in reasoned keyspace")
```

Comment:
this could either be interpreted as no answers are found (ie, answer size is 0), or that there are no inferred anwers that used reasoning (there may be answers that come form the DB. Let's make sure that the step and name of the methods being callsed are clear about the one thing they do :)

Reply:
We decided to delete the step, as it actually asserts the same thing as the step below it: `answer size in reasoned keyspace is: 0`.